### PR TITLE
Addition of the Work and About myself section for my portfolio

### PR DESCRIPTION
**Pull request by williamscch**
This pull request is to add the new parts to my portfolio on the project on day 4 at Microverse.
The new parts are "Works" section for works that I have done and the "About me" to show the knowledge and experience I have.
In order to layout the elements I used CSS Grid and Flexbox combined. CSS Grid in the main container that contains all of the cards and Flexbox to layout an individual card.
Implemented the use of the button interactions. And unordered lists to layout elements. 

I used template #3 on Figma.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,7 +45,174 @@ <h2 class="title2">I’m a software developer</h2>
...
+                        <li>Codekit</li>
+                        <li>GitHub</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </article>
```
Comment (tresorsawasawa) [inline index.html]: - [x] For kindly make sure all the containers of the **about me** section are wrapped in a `<section>` tag for goos semantics and Html best practice.

File: index.html [hunk 2]
```diff
@@ -45,7 +45,174 @@ <h2 class="title2">I’m a software developer</h2>
...
+                            <li>Css</li>
+                            <li>JavaScript</li>
+                            <li>Html</li>
+                        </ul>
+                    </div>
+                    <button type="button" value="See">See Project</button>
+                </div>
+            </div>
+        </div>
+    </article>
```
Comment (tresorsawasawa) [inline index.html]: - [x] Kindly make sure that you use the `<section>` tag to wrap all the cards container as required for the project and for good HTML semantics. Remember to change both open and close tags.

File: assets/css/style.css [hunk 3]
```diff
@@ -98,3 +99,178 @@ body {
...
+  background-color: var(--green);
+  color: var(--white);
+  border-color: var(--green);
+  border-radius: 5px;
+  width: 160px;
+  height: 50px;
+  font-weight: 500;
+  font-size: 17px;
+}
+
```
Comment (tresorsawasawa) [inline assets/css/style.css:230]: - [x] Kindly make sure the **Get My Resume** `button` follow the button styling: `Active`, `Hover`, and `Pressed` (see the design). Take a look below and check the correct colors from the Figma design:
buttons

File: index.html [hunk 4]
```diff
@@ -45,7 +45,174 @@ <h2 class="title2">I’m a software developer</h2>
...
+            what you see and have a project you need coded,
+            don’t hestiate to contact me.</p>
+        <button type="button" value="Get">Get My Resume</button>
+    </section>
+    <article class="my-skills">
+        <div class="skills-card">
+            <img src="./assets/img/icon-languages.svg"
+                alt="img">
+            <div class="skills-info">
+                <p>Languages</p>
```
Comment (tresorsawasawa) [inline index.html]: - [x] For HTML good semantics kindly use the `h` tag as this is a sub-title and not a paragraph. I can be **h2, h3, or h4**.

File: index.html [hunk 5]
```diff
@@ -45,7 +45,174 @@ <h2 class="title2">I’m a software developer</h2>
...
+                        <li>Html</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        <div class="skills-card">
+            <img src="./assets/img/icon-frameworks.svg"
+                alt="img">
+            <div class="skills-info">
+                <p>Frameworks</p>
```
Comment (tresorsawasawa) [inline index.html]: - [x] For HTML good semantics kindly use the `h` tag as this is a sub-title and not a paragraph. I can be **h2, h3 or h4**.

File: index.html [hunk 6]
```diff
@@ -45,7 +45,174 @@ <h2 class="title2">I’m a software developer</h2>
...
+                        <li>Selenium</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        <div class="skills-card">
+            <img src="./assets/img/icon-skills.svg"
+                alt="img">
+            <div class="skills-info">
+                <p>Skills</p>
```
Comment (tresorsawasawa) [inline index.html]: - [x] For HTML good semantics kindly use the `h` tag as this is a sub-title and not a paragraphh. I can be **h2, h3 or h4**.

File: index.html (hunk 1 above)
Comment (tresorsawasawa) [inline index.html]: - [x] Kindly make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `<ul>` tag, each of them in a `<li>`. And this `<ul>`  should be a part of **About section** not aoutside.

File: assets/css/style.css [hunk 7]
```diff
@@ -98,3 +99,178 @@ body {
...
+.product-info button {
+  background-color: var(--green);
+  color: var(--white);
+  border-color: var(--green);
+  border-radius: 5px;
+  width: 122px;
+  height: 50px;
+  font-weight: 500;
+  font-size: var(--md);
```
Comment (tresorsawasawa) [inline assets/css/style.css:153]: - [x] Kindly make sure the **Get My Resume** `button` follow the button styling: `Active`, `Hover`, and `Pressed` (see the design). Take a look at the previous pic where I pointed out the other button and check the correct colors from the Figma design.

File: assets/css/style.css [hunk 8]
```diff
@@ -98,3 +99,178 @@ body {
...
+  text-align: center;
+  margin: 0;
+}
+
+.about-me p {
+  font-size: var(--md);
+  text-align: center;
+  margin: 0 25px 0 25px;
+  font-weight: 400;
+}
```
Comment (tresorsawasawa) [inline assets/css/style.css]: - [x] Kindly make sure the about me description has the some `font-family`, `font-style`, `font-size`, `line-height`, and `color` as the one from [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988)

File: assets/css/style.css [hunk 9]
```diff
@@ -98,3 +99,178 @@ body {
...
+  height: 200px;
+}
+
+.product-info p {
+  margin: 0;
+  font-size: 25px;
+  font-weight: 600;
+  text-align: center;
+}
+
```
Comment (tresorsawasawa) [inline assets/css/style.css]: - [x] Make sure to add the correct used color from the Figma template design.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @wdavidcch 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 

## Required Changes :recycle: :recycle:
- [x] Kindly make sure to improve the design of your portfolio by correctly using the correct values provided at [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988) like:
   - Margin bottom of the **about me** section
   - And other comments pointed below
- [x] To make your README looks perfect with the required template for every project in Microverse, I suggest you correct the typo in its extension name. `README.md` instead of `README.me`.
- [x] Kindly improve your PR description to make it more explicit about all the details/changes you made in this milestone. I suggest you add on it a list of the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=Project%20requirements,enable%2C%20hover%2C%20etc.).) as they are changes you completed in the milestone
- [x] **Check the comments under the review**.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (williamscch) [reply index.html]: Hey I have to change the divs tag for section tags? or the article tag for section tag? @tresorsawasawa

File: index.html (hunk 2 above)
Comment (williamscch) [reply index.html]: are you talking about for example in line 139 change <div class="product-card> for <section class="product-card> @tresorsawasawa

File: index.html (hunk 1 above)
Comment (williamscch) [reply index.html]: Thank you... Can you be more specific I did not understand well what I have to change

Comment (williamscch) [reply index.html]: @tresorsawasawa

Comment (williamscch) [reply index.html]: Hey, be aware that I'm not in the 1st template so using a ul for the categories ("Languages", "Frameworks" and "Skills") maybe could not be the best way to do it in my template cuz they are separated like as cards. I'm in template 3.

If that is mandatory It's okay but please let me know before doing the new PR. @tresorsawasawa

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @wdavidcch 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passed ✔️ 
- Made most changes requested by previous reviewer 👍 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

[OPTIONAL]
- You added a few pseudo-classes required for actions on the buttons. Please add all of them as required by the Figma template.  As of now, when I hover over a button I see no change.  Follow this guide as seen below 
image

**Comment by williamscch** [comment]
Hey @Lucas-Masaba I noticed that but the hover is actually working, you can see it if you try with another color. What is happening here is that the color in the enable and in the hover is the same and we can not see the difference. Look it here...

enabled color: #36B37F
hover color: #36B37F

Screenshot from 2022-05-28 08-54-27
Screenshot from 2022-05-28 08-54-13

**Pull request merged**